main.go

```diff
@@ -96,21 +96,27 @@ func main() {
 	case "backup":
 		// TODO fix logic accordingly with apply-for
 		doBackup(serverInstance, applyFor, matchDashboard)
+		fmt.Println("Backup Complete.")
```

Comment:
Maybe better show these messages only in `verbose` mode? For CLI apps I try to stay with Unix ideology: utility should silently do it job and complain to stderr only on errors. So stdout should be used only for side effects (for example display listing) or in verbosity mode.

Reply:
Hrm... I'm two ways on that. You're right that a command line tool like this needs to play nicely with automation (that's pretty much exactly what it's for.) And all of the built in tools I can think of behave that way. However most of those all deal with local things and I've used a few of more complex tools which give you some indication by default.

That being said if we were to output a message by default we would want to add a --silent flag to make the thing shut up and that seems a little un-neccessary. I'll drop these messages and go back to the silent-by-default behavior when I update the PR for the above.